src/main/kotlin/build/buf/intellij/annotator/BufAnalyzePass.kt

```diff
@@ -273,14 +262,16 @@ fun AnnotationHolder.createAnnotationsForFile(
         .filter { it.path == file.virtualFile.toNioPath().relativeToOrNull(wd)?.toString() }
     for (issue in filteredIssues) {
         val severity = if (issue.isCompileError) HighlightSeverity.ERROR else HighlightSeverity.WARNING
-        val annotationBuilder = newAnnotation(severity, issue.message)
+        val type = if (severity == HighlightSeverity.ERROR) HighlightInfoType.ERROR else HighlightInfoType.WARNING
+        val highlightBuilder = HighlightInfo.newHighlightInfo(type)
+            .severity(severity)
+            .descriptionAndTooltip(issue.message)
             // TODO: Should handle file-level lint warnings here: https://github.com/bufbuild/intellij-buf/issues/215
             .range(issue.toTextRange(doc) ?: continue)
             .problemGroup { issue.type }
             .needsUpdateOnTyping(true)
-            .withFix(IgnoreBufIssueQuickFix(issue.type))
-
-        annotationBuilder.create()
+            .registerFix(IgnoreBufIssueQuickFix(issue.type), null, null, null, null)
```

Comment:
Didn't look in detail at the optional arguments here - verified that both linting and quick fixes work as they did before.